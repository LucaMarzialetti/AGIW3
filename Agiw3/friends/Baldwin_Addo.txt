--url_fb https://www.facebook.com/baldwin.r.addo
--friends
VM1823:3 Masaany Masaany
VM1823:3 Annmarie Addo
VM1823:3 Henry Addo
VM1823:3 Egwim Ignatius
VM1823:3 Owula Charles
VM1823:3 Geraldine Koduah
VM1823:3 Suelen Berdinase
VM1823:3 Naana Akosua Bridget
VM1823:3 Leonie Botchway
VM1823:3 Doe Felix
VM1823:3 Apostle Anna Marian Ardayfio
VM1823:3 Ogya Esi
VM1823:3 Raphael Thompson-addo
VM1823:3 Rita Yingi
VM1823:3 James Asante
VM1823:3 Nii Odartey Lamptey
VM1823:3 Nii Mo
VM1823:3 Lauren Addo
VM1823:3 Harribel Yankey
VM1823:3 Naa Shika
VM1823:3 Natasha Sylvia Thompson-Addo
VM1823:3 Baldwin Addo
VM1823:3 Charlie Khan
VM1823:3 Honza Sipek
VM1823:3 Fiifi Pratt
VM1823:3 LouLou G Crowe
VM1823:3 Dickson Anane-Adusei
VM1823:3 Reginald Thompson-addo
VM1823:3 Steffanny Lemos
VM1823:3 Rebecca Tetteh
VM1823:3 Richard Addo
VM1823:3 Oblietey Nii Obli
VM1823:3 Swee Lartey
VM1823:3 Xora Annette
VM1823:3 George Addo
VM1823:3 Samuel Donkor
VM1823:3 Stanley Botchway