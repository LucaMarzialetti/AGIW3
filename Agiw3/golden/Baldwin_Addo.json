{
    "PATTERN": {
        "tel": [
            "(516) 802-5388",
            "(516) 802-5388",
            "(516) 802-5388"
        ],
        "addr": ["839 Hastings St  Baldwin, New York 11510"],
        "email": []
    },
    "NER": {
        "LOC": [
            "New York",
            "Baldwin",
            "Atlanta",
            "Bronx",
            "Brooklyn",
            "Chicago",
            "Cleveland",
            "Dallas",
            "Denver",
            "Detroit",
            "Houston",
            "Las Vegas",
            "Los Angeles",
            "Miami",
            "Minneapolis",
            "New York",
            "Philadelphia",
            "Phoenix",
            "Saint Louis",
            "San Antonio",
            "San Diego",
            "San Jose"
        ],
        "ORG": [
            "Addo Trucking",
            "Addo Trucking Addo Trucking"
        ],
        "PER": []
    },
    "url": "https://www.chamberofcommerce.com/baldwin-ny/7150427-addo-trucking"
}